episodes/introduction_to_medical_imaging.md

```diff
@@ -65,102 +55,74 @@ associated with the lessons. They appear in the "Instructor View"
-```
-
-:::::::::::::::::::::::::::::::::
+Possible solutions include:
+- reviewing metadata on existing images so it can be matched by machine type
+- training technicians to use standardized settings only.
+- creating a machine set only on one power setting etc. 
+- OCR to get rid off burned in text
+- image harmonization/normalization algorithms 
 
 
 
-
-
-
-## Question 2: What are some disadvantages to ultrasound in terms of computational analysis?
-
-:::::::::::::::::::::::: solution 
-
-``Operator dependant, often with patient data burned in, settings vary, positions vary```
-
 :::::::::::::::::::::::::::::::::
 ::::::::::::::::::::::::::::::::::::::::::::::::
 
-## Figures
-
-You can use standard markdown for static figures with the following syntax:
-
-`![optional caption that appears below the figure](figure url){alt='alt text for
-accessibility purposes'}`
 
-![You belong in The Carpentries!](https://raw.githubusercontent.com/carpentries/logo/master/Badge_Carpentries.svg){alt='Blue Carpentries hex person logo with no text.'}
 
 ::::::::::::::::::::::::::::::::::::: callout
 
-Callout sections can highlight information.
+There is less standardization around file formats of certain kinds of imaging.
 
-They are sometimes used to emphasise particularly important points
-but are also used in some lessons to present "asides": 
-content that is not central to the narrative of the lesson,
-e.g. by providing the answer to a commonly-asked question.
+While typical radiological images have settled in how they are recorded in the DICOM
+even more novel sequences e.g. arterial spin labbeled sequences are not standardized in terms of how they are recoded in
+this file format. Pathology images are not neccesarily stored in DICOM at all, and there is
+some controversy as to what file format will come to dominate this important form of medical
+imaging. 
 
 ::::::::::::::::::::::::::::::::::::::::::::::::
 
 
-## Math
+## Visualization
 
-One of our episodes contains $\LaTeX$ equations when describing how to create
-dynamic reports with {knitr}, so we now use mathjax to describe this:
-
-`$\alpha = \dfrac{1}{(1 - \beta)^2}$` becomes: $\alpha = \dfrac{1}{(1 - \beta)^2}$
-
-Cool, right?
+There are increasing ways images can be visualized for human interpreters. 
+Optimizing visualization for humans and information for computer vision are
+very different processes. 
 
 ::::::::::::::::::::::::::::::::::::: keypoints 
 
-- Use `.md` files for episodes when you want static content
-- Use `.Rmd` files for episodes when you need to generate output
-- Run `sandpaper::check_lesson()` to identify any issues with your lesson
-- Run `sandpaper::build_lesson()` to preview your lesson locally
+- Every type and sequence of imaging gives you different and more information 
+- Computationally images have arrays but arrays are surrounded by other data
+- Research should be designed with human limitations in mind
+- We can expect more imaging modalities in the future
```

Comment:
Is anything about new possible imaging modalities mentioned in the text? I'd either mention it, or remove this key point